jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
+  collectCoverageFrom: [
+    '<rootDir>/src/**/*.{ts,tsx}',
+    "((\\.|/*.)(spec))\\.ts?$"
```

Comment:
```suggestion
    "((\\.|/*.)(spec))\\.ts?$"
    "((\\.|/*.)(spec))\\.tsx?$"
```

^ ou alguma forma de permitir ts e tsx